fcs/README.md

```diff
@@ -77,23 +74,21 @@ FAKE is only used to run build.fsx.  Eventually we will likely remove this once
 
 ### Testing
 
-Testing reuses the test files from ..\tests\service which were are also FCS tests. 
-
+Testing reuses the test files from ..\tests\service which were are also FCS tests.
 
 ### Documentation Generation
 
     fcs\build GenerateDocs
 
-Output is in ``docs``.  In the ``FSharp.Compiler.Service`` repo this is checked in and hosted as http://fsharp.github.io/FSharp.Compiler.Service.
-
+Output is in ``docs``.  In the ``FSharp.Compiler.Service`` repo this is checked in and hosted as <https://fsharp.github.io/FSharp.Compiler.Service>.
 
 ## The two other NuGet packages
 
-It also contains both the source, build, packaging and test logic for 
+It also contains both the source, build, packaging and test logic for
 
-* ``FSharp.Compiler.Service.MSBuild.v12`` adds legacy MSBuild v12 support to an instance of FSharp.Compiler.Service, if exact compatibility for scripting references such as ``#r "Foo, Version=1.3.4"`` is required. 
+- ``FSharp.Compiler.Service.MSBuild.v12`` adds legacy MSBuild v12 support to an instance of FSharp.Compiler.Service, if exact compatibility for scripting references such as ``#r "Foo, Version=1.3.4"`` is required.
```

Comment:
Why this change is needed?

Reply:
it was a complaint of mdlint in Visual Studio Code, where I accepted the proposed change to get rid of it.

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md004